workloads/kube-burner/README.md

```diff
@@ -68,7 +70,7 @@ Each iteration creates the following objects:
 
 The `node-density` and `node-density-heavy` workloads support the following environment variables:
 
-- **NODE_COUNT**: Number of worker nodes to deploy the pods on. During the workload nodes will be labeled with `node-density=true`. Defaults to 4.
+- **NODE_COUNT**: Number of worker nodes to deploy the pods on. During the workload nodes will be labeled with `node-density=true`. Defaults to the number of worker nodes across the cluster
```

Comment:
might want to note that we do not count nodes that are also marked as infra and workload in addition to worker

Reply:
fixed :)